boxsdk/object/file.py

```diff
@@ -791,9 +791,12 @@ def get_thumbnail_representation(self, dimensions, extension='png'):
             `bytes`
         """
         rep_hints = '[{0}?dimensions={1}]'.format(extension, dimensions)
-        representation = self.get_representation_info(rep_hints)
-        if representation:
-            url = representation[0]['content']['url_template']
+        representations = self.get_representation_info(rep_hints)
+        if len(representations):
+            representation = representations[0]
+            if representation['status'].get('code') in ('error_conversion_failed', 'error_password_protected'):
```

Comment:
Can you tell me where did you find information that we can return this error code? I was playing with the API and with encrypted PDF and I always got error_conversion_failed.

Reply:
ah, it was reported against a file named "Password protected.docx"